posthog/special_migrations/status.py

```diff
@@ -0,0 +1,15 @@
+def special_migrations_ok() -> bool:
+    from posthog.models.special_migration import MigrationStatus, SpecialMigration
+    from posthog.special_migrations.runner import is_migration_in_range
+
+    for migration in SpecialMigration.objects.all():
+        migration_completed_or_running = migration.status in [
```

Comment:
we added a delay before starting them automatically, so they would see this warning then, which is not ideal, not sure how to best solve that though

Reply:
good point

Reply:
for now, let's leave it - good way to make people aware of async migrations. one good thing to add is a pointer to user-facing docs from the special migrations UI